<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="RSI1525" name="br.com.rsinet.hub.projetoAppiumTDD.test.PesquisaHome" tests="3" failures="0" timestamp="2020-02-13T18:03:08 BRT" time="16.030" errors="1">
  <testcase name="testConfigsOff" time="7.993" classname="br.com.rsinet.hub.projetoAppiumTDD.test.PesquisaHome">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at br.com.rsinet.hub.projetoAppiumTDD.utility.Report.CloseTest(Report.java:65)
at br.com.rsinet.hub.projetoAppiumTDD.test.PesquisaHome.testConfigsOff(PesquisaHome.java:85)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:695)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:671)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:655)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testConfigsOff -->
  <system-out/>
  <testcase name="Pesquisar_Produto_Pela_Pagina_Inicial_Incorreto" time="0.000" classname="br.com.rsinet.hub.projetoAppiumTDD.test.PesquisaHome">
    <skipped/>
  </testcase> <!-- Pesquisar_Produto_Pela_Pagina_Inicial_Incorreto -->
  <system-out/>
  <testcase name="Pesquisar_Produto_Pela_Pagina_Inicial_Existente" time="8.037" classname="br.com.rsinet.hub.projetoAppiumTDD.test.PesquisaHome"/>
  <system-out/>
</testsuite> <!-- br.com.rsinet.hub.projetoAppiumTDD.test.PesquisaHome -->
